mentorship-platform/scss/style.scss

```diff
@@ -34,5 +45,207 @@ $white: white;
 .violet {
   background-color: $violet;
   margin: 16px 0;
+}
+
+.sections {
+  margin: 24px 5px;
+  padding: 30;
+  border: 1px solid;
+}
+
+.article {
+  padding: 5px;
+  margin: 20px;
+}
+
+.button {
+  margin: 5px;
+  padding: 5px;
+}
+
+.button--filled {
+  background-color: $background-color;
+}
+
+.button--outlined {
```

Comment:
Shorten the css code like
`.button {
&--filled {

}
&--outlined {

}... and so on
}
`